app/views/override/credits/credits-mow.md

```diff
@@ -0,0 +1,22 @@
+# Credits: March on Washington
+  
+The [March on Washington](/collections/march-march-on-washington) collection was supported in part by Save America's Treasures through a partnership between the National Endowment for the Arts and the National Park Service, Department of the Interior.
+
+<a href="http://www.arts.gov"><img height="80" src="https://s3.amazonaws.com/openvault.wgbh.org/logos/NEA.jpg"
```

Comment:
@mccalluc: This is the right logo. National Endowment for the Arts Saving America's Treasure's program.